app/views/reports/regions/details.html.erb

```diff
@@ -90,22 +96,24 @@
   <% end %>
 <% end %>
 
-<% if current_admin.permissions_v2_enabled? %>
-  <%= render 'shared/analytics_table',
-             dashboard_analytics: @dashboard_analytics,
-             period: @period.type,
-             row_resource: current_admin.accessible_facilities(:view_reports).order(:name),
-             row_resource_description: "Facilities",
-             row_resource_display_field: :name,
-             row_resource_link: :reports_region_facility_path %>
-<% else %>
-  <%= render 'shared/analytics_table',
-             dashboard_analytics: @dashboard_analytics,
-             period: @period.type,
-             row_resource: Facility.order(:name),
-             row_resource_description: "Facilities",
-             row_resource_display_field: :name,
-             row_resource_link: :reports_region_facility_path %>
+<% unless @region.is_a?(Facility) %>
+  <% if current_admin.permissions_v2_enabled? %>
+    <%= render "analytics/districts/analytics_table",
```

Comment:
@prabhanshuguptagit @harimohanraj89 I feel like I remember getting rid of this generic table in favor of two separate ones. Did this come back somehow?

Reply:
And if so, do we now have 3 of these? Generic, District-level, Facility-level?

Reply:
That was nice, but the reports re-use this markup now, so I re-partialized it.

There are indeed 3 versions now. I will trash the generic one (`shared/analytics_table`), it's not used.